README.md

```diff
@@ -29,23 +29,23 @@ module "public_subnets" {
-| `name`               |   ``    | Name  (e.g. `bastion` or `db`)                                                                                      |   Yes    |
-| `attributes`         |  `[]`   | Additional attributes (e.g. `policy` or `role`)                                                                     |    No    |
-| `tags`               |  `{}`   | Additional tags  (e.g. `map("BusinessUnit","XYZ")`                                                                  |    No    |
-| `names`              |   ``    | List of subnets names (e.g. `["apples", "oranges", "grapes"]`)                                                      |   Yes    |
-| `vpc_id`             |   ``    | The VPC ID where subnets will be created (e.g. `vpc-aceb2723`). If empty, a new VPC will be created                 |   Yes    |
-| `base_cidr`          |   ``    | The base CIDR block which will be divided into subnet CIDR blocks (e.g. `10.0.0.0/16`)                              |    No    |
-| `igw_id`             |   ``    | The Internet Gateway ID public route table will point to (e.g. `igw-9c26a123`). Conflicts with `nat_enabled = true` |    No    |
-| `nat_enabled`        | `false` | Flag to enable/disable NAT gateways. Conflicts with non-empty `igw_id`                                              |    No    |
-| `availability_zones` |   ``    | The  list of Availability Zones where subnets will be created (e.g. `["us-east-1a", "us-east-1b"]`).                |   Yes    |
-| `network_acl_id`     |   ``    | Network ACL ID that will be added to the subnets.  If empty, a new ACL will be created                              |    No    |
-| `additional_routes`  |  `{}`   | Map of Additional routes for subnets (e.g. `{"10.0.0.2/24" = igw-0038f979}`)                                        |    No    |
-| `egress`             |  `[]`   | List of Network ACL CIDRs permitted egress                                                                          |    No    |
-| `ingress`            |  `[]`   | List of Network ACL CIDRs permitted ingress                                                                         |    No    |
+| Name                  | Default | Description                                                                                                         | Required |
+|:----------------------|:-------:|:--------------------------------------------------------------------------------------------------------------------|:--------:|
+| `namespace`           |   ``    | Namespace (e.g. `cp` or `cloudposse`)                                                                               |   Yes    |
+| `stage`               |   ``    | Stage (e.g. `prod`, `dev`, `staging`)                                                                               |   Yes    |
+| `name`                |   ``    | Name  (e.g. `bastion` or `db`)                                                                                      |   Yes    |
```

Comment:
How is name used now? Seems like we should drop it.

Reply:
`name` is required field for `terraform-null-label`

Reply:
if you set name to `""` it will drop out.

Reply:
or actually, i would use `%v` and then it's a `format` template.